### PR TITLE
Ws mdms

### PR DESCRIPTION
**Pull request by shriya-egov-tarento**
Meter status json added and master config modified accordingly

### CODE REVIEW

File: data/pb/ws-services-calculation/meterStatus.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "tenantId": "pb",
+  "moduleName": "ws-services-calculation",
+  "meterStatus": [
```
Comment (nithindv) [inline data/pb/ws-services-calculation/meterStatus.json]: Could you make the master names sentence case? MeterStatus ?

File: data/pb/ws-services-calculation/meterStatus.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      "active": true
+    },
+    {
+      "name": "Breakdown",
+      "code": "BREAKDOWN",
+      "active": true
+    },
+    {
+      "name": "No-meter",
+      "code": "NOMETER",
```
Comment (nithindv) [inline data/pb/ws-services-calculation/meterStatus.json]: Use NO_METER if it makes sense in this context?

File: data/pb/ws-services-calculation/meterStatus.json (hunk 1 above)
Comment (shriya-egov-tarento) [reply data/pb/ws-services-calculation/meterStatus.json]: ok

File: data/pb/ws-services-calculation/meterStatus.json (hunk 2 above)
Comment (shriya-egov-tarento) [reply data/pb/ws-services-calculation/meterStatus.json]: ok

**Pull request merged**